runtime/src/weights/pallet_scheduler.rs

```diff
@@ -96,29 +96,29 @@ impl<T: frame_system::Config> pallet_scheduler::weights::WeightInfo for WeightIn
     // Storage: Scheduler Agenda (r:2 w:2)
     // Storage: Preimage PreimageFor (r:1 w:0)
     fn on_initialize_aborted(s: u32) -> Weight {
-        (7_540_000 as Weight)
-            // Standard Error: 141_000
-            .saturating_add((8_937_000 as Weight).saturating_mul(s as Weight))
+        (27_882_000 as Weight)
+            // Standard Error: 101_000
+            .saturating_add((8_796_000 as Weight).saturating_mul(s as Weight))
             .saturating_add(T::DbWeight::get().reads(2 as Weight))
             .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(s as Weight)))
             .saturating_add(T::DbWeight::get().writes(2 as Weight))
     }
     // Storage: Scheduler Agenda (r:2 w:2)
     // Storage: Scheduler Lookup (r:0 w:1)
     fn on_initialize_periodic_named(s: u32) -> Weight {
-        (45_414_000 as Weight)
-            // Standard Error: 131_000
-            .saturating_add((22_566_000 as Weight).saturating_mul(s as Weight))
+        (0 as Weight)
```

Comment:
Is this as expected? I checked, and this is non-zero in the weights.rs of the scheduler pallet 4.0.0.

Reply:
This is out of my control, sometimes the benchmarks for the same benchmark logic start with a base weight of greater than zero and sometimes it's zero. I think as long as the weight is modified properly by variables this is acceptable. If a change in the logic did not happen we can revert the base weight tough, I think it's better to overcharge than to undercharge (undercharging can lead to overweight blocks that can't be produced in time).